package.json

```diff
@@ -34,29 +34,26 @@
   ],
   "main": "lib/index.js",
   "dependencies": {
-    "canvas": "^1.6.7"
+    "canvas-prebuilt": "2.0.0-alpha.14"
   },
   "devDependencies": {
-    "coveralls": "^2.11.9",
-    "istanbul": "^0.4.3",
-    "jshint": "^2.9.2",
-    "mocha": "^3.0.2",
-    "mocha-lcov-reporter": "^1.2.0",
-    "mocha-phantomjs-core": "^2.0.1",
-    "phantomjs-prebuilt": "^2.1.15",
-    "uglify-js": "^2.6.2",
-    "unexpected": "^10.13.3"
+    "coveralls": "3.0.2",
+    "jshint": "2.9.6",
+    "mocha": "5.2.0",
+    "mocha-chrome": "1.1.0",
+    "nyc": "13.1.0",
+    "uglify-js": "3.4.9",
+    "unexpected": "10.39.1"
```

Comment:
Why are you switching to exact version numbers?

Reply:
Good question. Not intentionally